web/src/api/storage/types/config-model.ts

```diff
@@ -4,6 +4,14 @@
  * and run json-schema-to-typescript to regenerate this file.
  */
 
+export type EncryptionMethod =
+  | "luks1"
+  | "luks2"
+  | "pervasiveLuks2"
+  | "tpmFde"
+  | "protectedSwap"
```

Comment:
NP: more of the same. Several methods make little sense there. Not crucial since all this is temporary.